.editorconfig

```diff
@@ -1,9 +1,17 @@
 root = true
 
 [*]
-end_of_line = lf
-charset = utf-8
+end_of_line = lf
+charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
 indent_size = 4
+
+[*.json]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
```

Comment:
I think only the last line will be necessary within the *.json block, because the other values have benn already defined for all * files